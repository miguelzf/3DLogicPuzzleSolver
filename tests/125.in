X X X X -
X - - - -
X - L - -
X - - - -
X - - - B

P - - - P
X Y - - -
X - - - -
X - - - -
X X X X X

G - - - B
- L - - X
- - Y - X
- - - - X
G X X X X
