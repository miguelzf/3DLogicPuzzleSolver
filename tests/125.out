
Solution:
X  X  X  X  b  
X  p  p  p  b  
X  p  L  p  b  
X  p  l  p  b  
X  p  l  p  B  
=============
P  p  l  p  P  ||  G  g  g  g  B  
X  Y  l  l  l  ||  l  L  -  g  X  
X  y  -  -  y  ||  y  y  Y  g  X  
X  y  y  y  y  ||  g  g  g  g  X  
X  X  X  X  X  ||  G  X  X  X  X  
